ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java

```diff
@@ -33,7 +33,7 @@
     @Override
     public String render(final GetTapesWithFullDetailsResult obj) throws JsonProcessingException {
         final NamedDetailedTapeList result = obj.getTapesWithDetails();
-        if ((result == null) || (null == result.getNamedDetailedTapes())){
+        if ((result == null) || (result.getNamedDetailedTapes() == null) || (result.getNamedDetailedTapes().size() == 0 ) ){
```

Comment:
You should be able to use a Guard method to check if the List returned by getNamedDetailedTapes is null or empty.